cmd/privateactionrunner/subcommands/run/command.go

```diff
@@ -31,6 +35,21 @@ type cliParams struct {
 	*command.GlobalParams
 }
 
+func isPAREnabled(confFilePath string) bool {
+	// Create a minimal config instance to check PAR enablement
+	cfg := pkgconfigsetup.GlobalConfigBuilder()
+	cfg.SetConfigFile(confFilePath)
+
+	// Load config with minimal dependencies (no secrets, no custom loading)
+	err := pkgconfigsetup.LoadDatadog(cfg, secretsnoopimpl.NewComponent().Comp, nil)
+	if err != nil {
+		// If config loading fails, default to disabled
+		return false
+	}
+
+	return cfg.GetBool("privateactionrunner.enabled")
```

Comment:
nit-pick: I'm no yet familiar with the structure of the code base in this part of our system. However, it looks like it might be a good idea to have a single piece of logic that determines if PAR is enabled or not (i.e. reuse `IsEnabled`). Today is just a boolean in the config but perhaps it could become more complex later. Also, we would only need the code to know about the config entry name in a single place.